internal/store/elasticsearch/discovery_repository.go

```diff
@@ -179,3 +235,107 @@ func createUpsertBody(ast asset.Asset) (io.Reader, error) {
 
 	return &buf, nil
 }
+
+func (repo *DiscoveryRepository) clone(ctx context.Context, indexName, clonedIndexName string) error {
+	indexExistsFn := repo.cli.client.Indices.Exists
+	resp, _ := indexExistsFn([]string{clonedIndexName})
+	if resp.StatusCode == http.StatusOK {
+		return nil // skip clone when backup already created
+	}
+
+	cloneFn := repo.cli.client.Indices.Clone
```

Comment:
it's to make passing params easier in next line https://github.com/goto/compass/pull/68/files#diff-1d95d779f81167c420c72f47273dd57bd83290b18629f1b1194c17fbc3116fedR247

else it will look longer
resp, err := repo.cli.client.Indices.Clone(indexName, clonedIndexName, repo.cli.client.Indices.Clone.WithContext(ctx))